lib/gates_cirq.h

```diff
@@ -670,7 +750,10 @@ struct ZZPowGate {
   }
 };
 
-// The (exponent=1, global_shift=0) instance of XXPowGate.
+/**
+ * The `(exponent = 1, global_shift = 0)` instance of XXPowGate.
+ * This is the X-parity gate.
```

Comment:
One can write "The tensor product of two X gates" instead of "This is the X-parity gate", similarly for YY and ZZ.

Reply:
Done (also updated XX/YY/ZZPowGate).